book.bib

```diff
@@ -2439,23 +2439,24 @@ @misc{Vox_gun_violence
 @misc{Truman,
 author= {Harry Truman},
 title= {Harry Truman quote},
-year={2011},
-url={https://www.questia.com/read/1G1-263519053/history-in-the-making}
+year= {2011},
+url= {https://www.questia.com/read/1G1-263519053/history-in-the-making},
 }
 
-@misc{Tableau Calculated Fields,
```

Comment:
If you change the name of the reference in the bibliography, you will break the link when sited inline. Please make sure if you change a name like this you also go find the original inline reference and change it there as well.

Reply:
I agree. Looks like reference names were also changed in lines 2446, 2453 and 2675.

Reply:
Thank you for bringing this to my notice, fixing it now

Reply:
@gauravhassija I can see the reference name changes. Thanks!